src/Tribe/JSON_LD/Abstract.php

```diff
@@ -42,27 +49,47 @@ public static function instance( $name = null ) {
 
 	/**
 	 * Compile the schema.org event data into an array
+	 *
+	 * @param mixed $post            Either a post ID or a WP_post object.
+	 * @param array $args            {
+	 *                               Optional. An array of arguments to control the returned data.
```

Comment:
This looks just slightly weird. Can we format per [this example](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-1-parameters-that-are-arrays) (ie, indent the array args)?